[2025-01-10T16:43:04.598+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-10T16:43:04.610+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T16:42:00+00:00 [queued]>
[2025-01-10T16:43:04.617+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T16:42:00+00:00 [queued]>
[2025-01-10T16:43:04.617+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-10T16:43:04.629+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): append_to_database> on 2025-01-10 16:42:00+00:00
[2025-01-10T16:43:04.634+0000] {standard_task_runner.py:72} INFO - Started process 303 to run task
[2025-01-10T16:43:04.637+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'stock_information', 'append_to_database', 'scheduled__2025-01-10T16:42:00+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/vnstock_stream.py', '--cfg-path', '/tmp/tmpxva89d46']
[2025-01-10T16:43:04.639+0000] {standard_task_runner.py:105} INFO - Job 79: Subtask append_to_database
[2025-01-10T16:43:04.673+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_information.append_to_database scheduled__2025-01-10T16:42:00+00:00 [running]> on host 453ac4defedc
[2025-01-10T16:43:04.753+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='TuanDz' AIRFLOW_CTX_DAG_ID='stock_information' AIRFLOW_CTX_TASK_ID='append_to_database' AIRFLOW_CTX_EXECUTION_DATE='2025-01-10T16:42:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-10T16:42:00+00:00'
[2025-01-10T16:43:04.755+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-10T16:43:19.774+0000] {logging_mixin.py:190} INFO - Lỗi kết nối db: ('HYT00', '[HYT00] [Microsoft][ODBC Driver 18 for SQL Server]Login timeout expired (0) (SQLDriverConnect)')
[2025-01-10T16:43:19.777+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-10T16:43:19.787+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-10T16:43:19.790+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_information, task_id=append_to_database, run_id=scheduled__2025-01-10T16:42:00+00:00, execution_date=20250110T164200, start_date=20250110T164304, end_date=20250110T164319
[2025-01-10T16:43:19.835+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-10T16:43:19.855+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-10T16:43:19.859+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
